azure/services/fleetsmember/spec.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fleetsmember
+
+import (
+	"context"
+
+	asocontainerservicev1 "github.com/Azure/azure-service-operator/v2/api/containerservice/v1api20230315preview"
```

Comment:
Why is fleetmember in the same namespace as AKS in ASO? that seems wrong? We release with a different SDK, different API version, etc...
our latest API version is 0815preview.

Reply:
Hmm I'm not sure. ASO includes both under asocontainerservicev1 here: https://azure.github.io/azure-service-operator/reference/containerservice/